internal/service/federatedsettingsorgconfig/resource_federated_settings_connected_org_test.go

```diff
@@ -105,13 +128,37 @@ func importStateIDFunc(federationSettingsID, orgID string) resource.ImportStateI
 	}
 }
 
-func configBasic(federationSettingsID, orgID, identityProviderID, associatedDomain string) string {
+func configBasic(federationSettingsID, orgID, identityProviderID, associatedDomain string, withWorkload, useWorkload, domainRestrictionEnabled bool) string {
```

Comment:
can we rename `withWorkload` and `useWorkload`? It's not super clear to me what the difference is by these names

Reply:
(hopefull) improved in d7630abd3e26cf33e7afbc149137d22bd9b87a14 😅 
WDYT?